webtester-support-junit5/src/main/java/info/novatec/testit/webtester/junit5/exceptions/NoConfigurationUnmarshallerFoundException.java

```diff
@@ -0,0 +1,26 @@
+package info.novatec.testit.webtester.junit5.exceptions;
+
+import lombok.Getter;
+
+import info.novatec.testit.webtester.junit5.extensions.configuration.ConfigurationValue;
```

Comment:
It's not really unused ... The import exists because of the JavaDoc mention of the ConfigurationValue annotation.

@bitterblue This is actually an interesting topic for a discussion:
What's the (hidden) consequence of importing a class?

In this instance I would say this exception class belongs in the package with the annotation it is referring to. Not actually using the class in code but in the documentation tells me it belongs to the other classes on a functional level.